src/platform/forms-system/src/js/actions.js

```diff
@@ -228,19 +229,25 @@ export function uploadFile(
       uiOptions.maxSize;
 
     if (file.size > maxSize) {
+      const fileSizeText = displayFileSize(maxSize);
       onChange({
         name: file.name,
-        errorMessage: 'File is too large to be uploaded',
+        errorMessage:
+          'We couldn\u2019t upload your file because it\u2019s too big. ' +
+          `Please delete this file. Then upload a file that\u2019s ${fileSizeText} or less.`,
```

Comment:
Has someone from content reviewed these message changes? cc @peggygannon

Reply:
I believe @DanielleThierryUSDSVA  worked on this with our designer Dene.

Reply:
I did work on these with Dené, yes. Thanks, @dat-gcio. 

@peggygannon, here's the content we worked out. Let me know if you have any questions. I'll also ask Randi to update in the content style guide. Thanks!

**Network request failed**

We're sorry. We had a connection problem. Please delete the file and try again.
[Delete file]

**File too large**
We couldn't upload your file because it's too big. Please delete this file. Then upload a file that's 10MB or less.
[Delete file]

**File too small**
We couldn’t upload your file because it’s too small. Please delete this file. Then upload a file that’s 1B or more.
[Delete file]

**File not allowed type**
We couldn't upload your file because we can't accept this type of file. Please delete the file. Then try again with a .pdf, .jpeg, .jpg, or .png file.
[Delete file]

Reply:
@DanielleSoCompany OK, thank you for the update.

Reply:
@peggygannon @DanielleThierryUSDSVA Are we good to move forward, or do we need to wait on additional approval?